### PR TITLE
New targets

### PR DESCRIPTION
**Pull request by Vsevolod-Livinskij**
Add new targets: Sapphire Rapids and Polly. The default binutils do not fully support spr, so we have to build the latest version by hand.

### CODE REVIEW

File: scripts/gen_test_makefile.py [hunk 1]
```diff
@@ -118,6 +118,7 @@ def __init__(self, name, enum_value):
 SdeArch["skx"] = SdeTarget("skx", 10)
 SdeArch["icx"] = SdeTarget("icx", 11)
 SdeArch["tgl"] = SdeTarget("tgl", 12)
+SdeArch["spr"] = SdeTarget("spr", 12)
```
Comment (dbabokin) [inline scripts/gen_test_makefile.py:121]: Copy-paste error: 12!

File: docker/Dockerfile [hunk 2]
```diff
@@ -40,9 +49,11 @@ RUN git clone $YARPGEN_REPO $YARPGEN_HOME &&\
 
 RUN $YARPGEN_HOME/scripts/build_gcc.sh $GCC_REPO $GCC_VERSION
 
+RUN $YARPGEN_HOME/scripts/build_binutils.sh $BINUTILS_REPO
```
Comment (dbabokin) [inline docker/Dockerfile:52]: Need a comment why it's needed

File: scripts/gen_test_makefile.py (hunk 1 above)
Comment (Vsevolod-Livinskij) [reply scripts/gen_test_makefile.py]: Fixed

File: docker/Dockerfile (hunk 2 above)
Comment (Vsevolod-Livinskij) [reply docker/Dockerfile:56]: Fixed

**Pull request merged**